lib/ejs.js

```diff
@@ -84,17 +84,18 @@ exports.localsName = _DEFAULT_LOCALS_NAME;
  * Get the path to the included file from the parent file path and the
  * specified path.
  *
- * @param {String} name     specified path
- * @param {String} filename parent file path
+ * @param {String}  name     specified path
+ * @param {String}  filename parent file path
+ * @param {Boolean} isdir    parent file path whether is directory
```

Comment:
Just to clarify what @mde is saying, `isdir` should be renamed to `isDir` or `isDirectory`.